go/worker/compute/executor/committee/node.go

```diff
@@ -792,7 +756,19 @@ func (n *Node) proposeTimeoutLocked(roundCtx context.Context) error {
 
 		// Make sure we are still in the right state/round.
 		n.commonNode.CrossNode.Lock()
-		if _, ok := n.state.(StateWaitingForBatch); !ok || round != n.commonNode.CurrentBlock.Header.Round {
+		// Make sure we are still in the right state.
+		var invalidState bool
+		switch n.state.(type) {
+		case StateWaitingForBatch:
```

Comment:
Merge these two cases (style), unless you think there will be a need to split them or add explicit coments.